tests/integration/ceph_flex_test.go

```diff
@@ -92,7 +92,7 @@ func (s *CephFlexDriverSuite) SetupSuite() {
 		usePVC:                  false,
 		mons:                    1,
 		rbdMirrorWorkers:        1,
-		rookCephCleanup:         false,
+		rookCephCleanup:         true,
```

Comment:
Why aren't all the tests cleaning up after themselves now? We don't want to cause dependency issues between  the test suites.

Reply:
I don't remember but purging every time did not work, they might be legacy reasons not to purge all the time. Honestly, I don't know.